### ISSUE TITLE
IFD error while deploying from x86_64 to arm64 with ` --build-on-target`

### ISSUE THREAD

**Issue by shokinn** [nix-community/authentik-nix#5]
Hi,
I got some IFD issue while deploying with colmena from my local machine (x86_64) to an arm64 based server.

A friend of mine helped me to get this sorted, and since I'm actually still don't understand what she did, it works now.
But because of that I'm not comfortable to file a PR.
The changes she suggested are the following:
https://github.com/nix-community/authentik-nix/compare/main...shokinn:authentik-nix:main

I kindly like to ask to add those to your repo as well.

**Comment by WilliButz** [comment]
Hey, thanks for pointing that out. I'm currently not working with the limitation of `allow-import-from-derivation = false`, so I did not notice that I introduced it, but I think it should be avoided if possible.
Not sure how [that additional cargo-hash](https://github.com/nix-community/authentik-nix/commit/00435aafc10890636307aca4146e9682e515f03a) avoids IFD but the patched pyproject.toml is obvious.
